### PR TITLE
Update for 0.85.2

### PR DESCRIPTION
**Pull request by celandro**

### CODE REVIEW

File: versions/latest-version.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.83.3
+0.85.2.1
```
Comment (BrunnerLivio) [inline versions/latest-version.txt]: The version is going backwards?

**Comment by celandro** [comment]
There were 2 different 0.85.2 protos. One with a buffed Ho-Oh and then a
few hours later nerfed.  Not sure what to do with naming convention here

**Comment by BrunnerLivio** [comment]
The problem is 0.85.2.1 does not follow the [Semantic Versioning guidelines](https://semver.org/). I would add the releasedate of the buffed version to the version name.

E.g:

`0.85.2.1` -> `0.85.2+20171210132500` for 2017/12/10 13:25:00. This is a valid semantic version and Travis would not fail then.

Edit:
Yeah I think 0.85.2-1 is better than the timestamp version

**Comment by Drexlin** [comment]
Hey guys,

Sorry, I'm new here. Any idea when the new json file will be available in
the "latest" directory? I use that to update my database.

Thanks!
Brian

File: versions/latest-version.txt (hunk 1 above)
Comment (goatslacker) [reply versions/latest-version.txt]: Looks like it's going forward but it's not valid SemVer.

If you can trim it down to `0.85.2` the tests will pass. Or `0.85.2-1` works too.

**Comment by celandro** [comment]
I just pushed today's. I'm very busy with the site. Decide the exact
directory scheme you want and I will fix tonight

**Comment by dandesousa** [comment]
My opinion for what its worth: since its not uncommon to update GM files more than once per version, I think a standard convention of `0.85.2-(\d)+` (required number starting at -1) is best going forward. So this can have `0.85.2-1` and `0.85.2-2`.

**Comment by dandesousa** [comment]
@celandro  Feel free to merge https://github.com/BrunnerLivio/pokemongo-game-master/pull/28 into this branch, it includes all the version changes and a small update to the README to clarifying versioning going forward.

Hopefully it saves you some time.

**Review by BrunnerLivio — APPROVED** [review]

**Comment by BrunnerLivio** [comment]
I think we can merge this one, or is anything pending @celandro ?

**Review by goatslacker — APPROVED** [review]

**Review by celandro — COMMENTED** [review]
lgtm

**Pull request merged**